projects/plugins/jetpack/extensions/blocks/premium-content/index.js

```diff
@@ -12,12 +12,28 @@ import save from './save';
 import icon from './_inc/icon';
 import { blockContainsPremiumBlock, blockHasParentPremiumBlock } from './_inc/premium';
 
+/**
+ * A list of blocks that should be disallowed to be transformed to Premium content block since they are mostly markup blocks.
+ *
+ * @type {{"core/loginout": boolean, "core/nextpage": boolean, "core/more": boolean, "core/separator": boolean, "core/spacer": boolean, "core/post-navigation-link": boolean}}
+ */
+const disallowFromTransformations = {
```

Comment:
Given that you're just checking for presence in a list, the values are meaningless, this could've just been an array using a `.includes` rather than a `.hasOwnProperty`. 

It makes little difference though.